app/Socket/Pusher.php

```diff
@@ -37,15 +37,17 @@ static function sendDataToServer(array $data)
     public function broadcastData($jsonDataToSend)
     {
         $entryData = json_decode($jsonDataToSend, true);
-
         $subscribedTopics = $this->getSubscribedTopics();
 
+        if (!isset($entryData['topic_id']) || empty($entryData['topic_id'])) {
+            return;
```

Comment:
I think empty check should be enough here. Also please log this kind of situation in order to detect what caused this situation.